test/check/scrive_api.md

```diff
@@ -5416,7 +5416,7 @@ Similarly, a single field can have multiple placements on the document.</p>
 <p><strong>Note:</strong> Some field types have <em>no effect</em> without at least one placement.</p>
 
 
-<p>The value of this array must match <em>at least one</em> of the following schemas:</p>
+<p>Each element of this array must match <em>at least one</em> of the following schemas:</p>
```

Comment:
This is unrelated to this PR. It should go away if you merge https://github.com/scrive/openapi2slate/pull/16 first though.

Reply:
Exactly. Created the PR branch as a "spinoff" branch to avoid having to merge/rebase later.